### PR TITLE
Hamnanimra5110/ch71/model tests

### PR DESCRIPTION
**Pull request by HamnaNimra**
Entity tests

### CODE REVIEW

File: src/test/java/Sysc4806Group/demo/entityTests/BookTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+    Book book;
+
+    @org.junit.jupiter.api.BeforeEach
+    void setUp() {
+        this.book = new Book( "123456", "XYZ Book", "Random Author", "Nelson", "google.com/images", 15);
+    }
+
+    @org.junit.jupiter.api.AfterEach
+    void tearDown() {
```
Comment (SVD6) [inline src/test/java/Sysc4806Group/demo/entityTests/BookTest.java]: What's this?

**Review by SVD6 — CHANGES_REQUESTED** [review]
Add one/two non assertequals tests, just to add more tests to the bunch

File: src/test/java/Sysc4806Group/demo/entityTests/BookTest.java (hunk 1 above)
Comment (HamnaNimra) [reply src/test/java/Sysc4806Group/demo/entityTests/BookTest.java]: > Add one/two non assertequals tests, just to add more tests to the bunch

Makes sense, will do that

Comment (HamnaNimra) [reply src/test/java/Sysc4806Group/demo/entityTests/BookTest.java]: > What's this?

It doesn't do much right now so I can remove it but was hoping to use it later on

**Review by SVD6 — APPROVED** [review]

**Pull request merged**